tests/e2e/tests/stackdriver/adapter.yaml

```diff
@@ -0,0 +1,237 @@
+apiVersion: "config.istio.io/v1alpha2"
+kind: stackdriver
```

Comment:
can we use the OOP-style config here? Something like:

```yaml
kind: handler
metadata:
  name: stackdriver
spec:
  compiledAdapter: stackdriver
  params:
    appCredentials: y
    ...
```

Reply:
Done